### PR TITLE
Load function

### PR DESCRIPTION
**Pull request by bananenpampe**
Adds a load function to load the .json or .json.gz file after saving a jupyter widget

### CODE REVIEW

**Comment by ceriottm** [comment]
Looks good but I fear the semantics is a bit confusing. `save` is a method associated with a chemiscope widget, that saves its status. `load` should be similarly a method that loads a ".json" into an existing chemiscope widget. 
IDK if this is something that makes sense, and/or something you want to implement in this PR but I would rename the current `load` method as something like `read_input` or probably `show_input`

File: python/chemiscope/jupyter.py [hunk 1]
```diff
@@ -101,6 +101,64 @@ class MapWidget(ChemiscopeWidgetBase):
     def __init__(self, data, has_metadata):
         super().__init__(data, has_metadata)
 
+def load(path, mode="default"):
```
Comment (ceriottm) [inline python/chemiscope/jupyter.py:105]: I would rename this as show_input

File: python/chemiscope/jupyter.py [hunk 2]
```diff
@@ -101,6 +101,64 @@ class MapWidget(ChemiscopeWidgetBase):
...
+    """
+    Loads a dataset from given ``path``.
```
Comment (ceriottm) [inline python/chemiscope/jupyter.py:107]: I would indicate "Loads and show as a chemiscope widget" ...

**Review by ceriottm — CHANGES_REQUESTED** [review]
I would also add something to the documentation.

**Comment by bananenpampe** [comment]
It makes sense, yet it would require first initializing an empty widget and then loading the saved widget. I dont really see a usecase for it. I changed the names.

**Comment by ceriottm** [comment]
> It makes sense, yet it would require first initializing an empty widget and then loading the saved widget into the empty one. I dont really see a usecase for it. I changed the names.

I am thinking of having a loaded CS, and then changing the file it has loaded. Might be interesting in interactive notebooks. I think however it's a corner case, and we could cover it in another PR if the need ever arises.

**Review by ceriottm — APPROVED** [review]

**Pull request merged**